src/store/constants/index.js

```diff
@@ -1,4 +1,2 @@
 export const ADD_TASK = "ADD_TASK";
-export const DELETE_TASK = "DELETE_TASK";
-export const COMPLETE_TASK = "COMPLETE_TASK";
-export const EDIT_TASK = "COMPLETE_TASK";
+export const CREATE_BUTTON = "CREATE_BUTTON";
```

Comment:
Can we write the const with more understandable way like the top one ? Example: `TOGGlE_CREATE_BUTTON_VISIBILITY`